File: Solutions/3414/c++/program.cpp
// Copy from Quera

// In the name of God

#include <iostream>
using namespace std;

int main()
{
    int x1, y1, x2, y2;
    cin >> x1 >> y1 >> x2 >> y2;

    if (x1 == x2)
        cout << "Vertical\n";
    else if (y1 == y2)
        cout << "Horizontal\n";
    else
        cout << "Try again\n";
    return 0;
}

File: Solutions/3414/java/program.java
// Copy from Quera

import java.util.Scanner;

public class p2q2 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int x0 = scanner.nextInt(), y0 = scanner.nextInt(), x1 = scanner.nextInt(), y1 = scanner.nextInt();
        if(x0 == x1)
            System.out.println("Vertical");
        else if(y0 == y1)
            System.out.println("Horizontal");
        else
            System.out.println("Try again");
    }
}
